api/app/Models/Pool.php

```diff
@@ -314,8 +314,139 @@ public static function scopeCurrentlyActive(Builder $query)
         return $query;
     }
 
+    public static function scopeName(Builder $query, ?string $name): Builder
+    {
+        if ($name) {
+            $query->where(function ($query) use ($name) {
+                $term = sprintf('%%%s%%', $name);
```

Comment:
What exactly to the five % do? It thought we usually only have one on either side 😆

Reply:
with `sprintf` the `%` indicates a replacement value so you add two `%%` to escape it :sweat_smile:

Reply:
https://www.php.net/manual/en/function.sprintf.php#:~:text=With%20printf()%20and%20sprintf()%20functions%2C%20escape%20character%20is%20not%20backslash%20%27%5C%27%20but%20rather%20%27%25%27.